build-run-docker.sh

```diff
@@ -0,0 +1,28 @@
+# build the ess website docker image and run it in a container (locally)
```

Comment:
Ich bin im allgemeinen etwas verhalten bei solchen Helferskripten, da jeder das ein wenig anders startet/nutzt. *Wie* es prinzipiell gestartet werden kann, steht auch schon in der Readme, d.h. der dokumentatorische Aspekt ist auch gering.

Ansonsten fehlt in der ersten Zeile noch das Shebang:
```suggestion
#!/bin/sh
# build the ess website docker image and run it in a container (locally)
```